docs/reference/units/job.rst

```diff
@@ -139,6 +139,14 @@ Following fields may be used by the job unit:
     run (regardless of the outcome). Multiple job names, separated by spaces,
     can be specified.
 
+.. option:: before
```

Comment:
There should probably be some kind of warning when using this field alongside the `also-after-suspend` flag, as this field will also be present in the generated sibling, which means any job listed in the `before` field will be pushed much later in the test plan, which might not be the desired behavior!